app/src/main/java/com/framgia/soundclound/screen/moretrack/MoreTrackViewModel.java

```diff
@@ -1,7 +1,12 @@
 package com.framgia.soundclound.screen.moretrack;
 
+import android.Manifest;
+import android.app.Activity;
 import android.content.Context;
+import android.content.pm.PackageManager;
 import android.databinding.BaseObservable;
+import android.support.v4.app.ActivityCompat;
```

Comment:
ERROR: Unused import: android.support.v4.app.ActivityCompat. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports